protostar/commands/cairo1_commands/cairo1_build_command.py

```diff
@@ -0,0 +1,79 @@
+import logging
+from pathlib import Path
+from typing import List, Optional, Any
+
+from protostar.cli import ProtostarArgument, ProtostarCommand
+from protostar.cli.common_arguments import COMPILED_CONTRACTS_DIR_ARG
+from protostar.compiler import (
+    ProjectCompiler,
+)
+import protostar.cairo.cairo_bindings as cairo1
+
+
+class Cairo1BuildCommand(ProtostarCommand):
+    def __init__(
+        self,
+        project_compiler: ProjectCompiler,
```

Comment:
It seems weird that we use a compiler from previous versions here. Can't we make it separate, or pass higher level objects?

Reply:
i think this point still stands (after discussion about interop)

Reply:
[avoid passing proj compiler to the build command](https://github.com/software-mansion/protostar/pull/1551/commits/2ae4aa91169bacb80b98de40fee8fe1799529653) please let me know if that's ok